addon/src/test-support/helpers/drag.js

```diff
@@ -108,4 +108,11 @@ export async function drag(mode, itemSelector, offsetFn, callbacks = {}) {
     await callbacks.dragend();
     await settled();
   }
+
+  await waitUntil(
+    () => {
+      return !find('.is-dropping');
+    },
+    { timeout: 2000 }
+  );
```

Comment:
This helps us to fix tests, because in tests there is no point in which we do wait for `transitionEnd`. This bug is possible to reproduce, when transition time is high or when someone has a lot of items

Reply:
this change was moved to PR #562